jetty-http/src/test/java/org/eclipse/jetty/http/HttpParserTest.java

```diff
@@ -286,7 +287,7 @@ public void testFoldedField7230() throws Exception
         Assert.assertThat(_bad, Matchers.containsString("Header Folding"));
         Assert.assertNull(_complianceViolation);
     }
-    
+
```

Comment:
while this is a good formatting change... I really don't like seeing unrelated reformatting in PRs of substance.

Reply:
Sorry, I thought I have removed all unnecessary code reformattings.